docs/cennznet/rpc.md

```diff
@@ -0,0 +1,370 @@
+## JSON-RPC
+
```

Comment:
Auto generated rpc (from definitions.ts)
Also includes genericAssets and Cennzx RPC calls